content/docs/tasks/traffic-management/egress-gateway.md

```diff
@@ -0,0 +1,317 @@
+    which will be used as a test source for external calls.
+
+    If you have enabled [automatic sidecar injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection), do
+
+    ```command
+    $ kubectl apply -f @samples/sleep/sleep.yaml@
+    ```
+    otherwise, you have to manually inject the sidecar before deploying the `sleep` application:
+
+    ```command
+    $ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@)
+    ```
+
+    Note that any pod that you can `exec` and `curl` from would do.
+
+*   Create a shell variable to hold the name of the source pod for sending requests to external services.
+If we used the [sleep](https://github.com/istio/istio/tree/{{<branch_name>}}/samples/sleep) sample, we run:
+    ```command
+    $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})
+    ```
+
+## Define an egress `Gateway` and direct HTTP traffic through it
+
+First let's direct HTTP traffic without TLS origination
+
+1.  Create an egress `Gateway` for _edition.cnn.com_, port 80:
+
+    ```bash
+        cat <<EOF | istioctl create -f -
+        kind: Gateway
+        metadata:
+          name: istio-egressgateway
+        spec:
+          selector:
+            istio: egressgateway
+          servers:
+          - port:
+              number: 80
+              name: http
+              protocol: HTTP
+            hosts:
+            - "edition.cnn.com"
+    EOF
+    ```
+
+1.  Define a `ServiceEntry` for `edition.cnn.com` and a `VirtualService` to direct
+the traffic through the egress gateway:
+
+    ```bash
+        cat <<EOF | istioctl create -f -
+        apiVersion: networking.istio.io/v1alpha3
+        kind: ServiceEntry
+        metadata:
+          name: cnn
+        spec:
+          hosts:
+          - edition.cnn.com
+          ports:
+          - number: 80
+            name: http-port
+            protocol: HTTP
+          - number: 443
+            name: https
+            protocol: HTTPS
+          resolution: DNS
+        ---
+        apiVersion: networking.istio.io/v1alpha3
+        kind: VirtualService
+        metadata:
+          name: direct-through-egress-gateway
+        spec:
+          hosts:
+          - edition.cnn.com
+          gateways:
+          - istio-egressgateway
+          - mesh
+          http:
+          - match:
+            - gateways:
+              - mesh
+              port: 80
+            route:
+            - destination:
+                host: istio-egressgateway.istio-system.svc.cluster.local
+                port:
+                  number: 80
+              weight: 100
+          - match:
+            - gateways:
+              - istio-egressgateway
+              port: 80
+            route:
+            - destination:
+                host: edition.cnn.com
+                port:
+                  number: 80
+              weight: 100
+        EOF
```

Comment:
This need to be align with "```"

Reply:
I think it has to be aligned with the first command, `cat` in this case. I see it in other tasks too.